rustica/src/auth/external.rs

```diff
@@ -270,9 +270,7 @@ impl AuthServer {
         auth_props: &X509AuthorizationRequestProperties,
     ) -> Result<X509Authorization, AuthorizationError> {
         let mut authorization_request = HashMap::new();
-        // TODO: This is an anachronistic hold over and should be updated from
-        // quorum_mtls to rustica_mtls or rustica_x509
-        authorization_request.insert(format!("type"), "quorum_mtls".to_string());
+        authorization_request.insert(format!("type"), "rustica_mtls".to_string());
```

Comment:
This is a breaking but needed change.

Reply:
The authorizer we have handles both `rustica_mtls` and `quorum_mtls` so it's not breaking for our stack.